src/compiler/typescript_to_rust/generators/ic_object/functions/reject.ts

```diff
@@ -0,0 +1,16 @@
+import { Rust } from '../../../../../types';
+
+export function generateIcObjectFunctionReject(): Rust {
+    return /* rust */ `
+        fn _azle_ic_reject(
+            _this: &boa_engine::JsValue,
+            _aargs: &[boa_engine::JsValue],
+            _context: &mut boa_engine::Context,
+        ) -> boa_engine::JsResult<boa_engine::JsValue> {
+            Ok(
+                ic_cdk::api::call::reject(_aargs.get(0).unwrap().as_string().unwrap())
```

Comment:
azle_try_from_js_value

Reply:
I'm probably just doing it wrong, but I'm not able to get it to work.

`__args.get(0)` is an `Option` so you have to `.unwrap()` to get the value, but then I get this:

![Screenshot from 2022-06-24 17-04-47](https://user-images.githubusercontent.com/5455419/175721764-91aa620a-ced9-4f54-a140-d54c254fb463.png)

...I'm not sure how to get this working right.

Reply:
azle_try_from_js_value returns a result, so you need to unwrap it

Reply:
So do: `_aargs.get(0).unwrap().azle_try_from_js_value().unwrap()`